hail/src/main/scala/is/hail/expr/ir/Parser.scala

```diff
@@ -968,6 +968,9 @@ object IRParser {
       case "MatrixCollectColsByKey" =>
         val child = matrix_ir(env)(it)
         MatrixCollectColsByKey(child)
+      case "MatrixRepartition" =>
+        val child = matrix_ir(env)(it)
+        MatrixRepartition(child, int32_literal(it), boolean_literal(it))
```

Comment:
Does Scala guarantee the order the arguments are executed? I'd prefer they were explicitly stated like `child`.

Reply:
scala does evaluate arguments left-to-right, but I'm happy to mimic the above